packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -19,6 +26,8 @@ export interface ChatMessageActionFlyoutProps {
   onEditClick?: () => void;
   onRemoveClick?: () => void;
   onDismiss: () => void;
+  messageThreadReadCount?: number;
```

Comment:
I find this prop name a little misleading. It's how many participants have read the message right? It isn't related to the "thread" per say?

How about `messageReadByCount` ?

Reply:
updated :)